common/system/filtering/filters.h

```diff
@@ -79,6 +79,14 @@ int8_t filter_getGravityRef();
  */
 FilterData_s* filter_getData();
 
+/**
+ * @brief Sets gyro offsets
+ * @param xOffset: x-axis gyro offset
+ * @param yOffset: y-axis gyro offset
```

Comment:
Do we have a comment anywhere saying what the offsets do, and what units they're in? I assume rad/s but it might be good to note